geniza/corpus/metadata_export.py

```diff
@@ -165,3 +172,14 @@ def get_export_data_dict(self, doc):
         outd["has_translation"] = doc.has_translation()
 
         return outd
+
+
+class AdminDocumentExporter(DocumentExporter):
+    exclude_csv_fields = set()
+
+
+class PublicDocumentExporter(DocumentExporter):
+    exclude_csv_fields = {"notes", "needs_review", "url_admin", "status"}
```

Comment:
can we just set `csv_fields` to the base list of fields minus these? I'm not sure what the exclude property gets us